src/common/types.ts

```diff
@@ -0,0 +1,12 @@
+import { ProductMeta, Products } from '@hashicorp/platform-product-meta'
```

Comment:
Bike-shedding a bit here....but I would maybe prefer a `src/types` directory and then expose meaningful submodules, e.g. for this stuff `src/types/products.ts` feels more intuitive.

Some immediate thoughts that led me here: What else would go in common? What if this types file gets really big? What if we want to create separate types files just to make it a little more discoverable?

Reply:
I'm cool with this, I'll make the change! I really like your point about discoverability.